src/components/Chat/Chat.tsx

```diff
@@ -170,44 +179,38 @@ export const Chat = memo(() => {
     [dispatch],
   );
 
-  useEffect(() => {
-    if (!autoScrollEnabled || !chatContainerRef.current) {
-      return;
-    }
+  const setAutoScroll = () => {
+    clearTimeout(disableAutoScrollTimeoutRef.current);
+    setAutoScrollEnabled(true);
+    setShowScrollDownButton(false);
+  };
 
-    chatContainerRef.current.scrollTo({
-      top: chatContainerRef.current.scrollHeight,
-      behavior: 'smooth',
-    });
-    textareaRef.current?.focus();
-  }, [autoScrollEnabled]);
+  const scrollDown = useCallback(
+    (force = false) => {
+      if (autoScrollEnabled || force) {
+        setAutoScroll();
+        messagesEndRef.current?.scrollIntoView({
```

Comment:
Please check how this is working in overlay, because when we previously use this we had an issue that parent window also scrolled , but scrollTo is working fine with that

Reply:
Thank you. Yes, issue was reproducible in overlay mode. Fixed and pushed